.github/workflows/classify.yaml

```diff
@@ -53,3 +51,16 @@ jobs:
       - uses: actions-ecosystem/action-add-labels@v1
         with:
           labels: kind/${{ steps.regex-match.outputs.group1 }}
+
+      - name: Check release notes were not deleted
```

Comment:
nit: isn't this more checking that they are present?

Reply:
This is philosophical :P 
The PR template now has the section, so the only reason for the section not be there is if the contributor deleted it